packages/framework/composer.json

```diff
@@ -51,6 +51,7 @@
         "fp/jsformvalidator-bundle": "^1.5.1",
         "fzaninotto/faker": "^1.7.1",
         "gedmo/doctrine-extensions": "^2.4.34",
+        "google/cloud-storage": "^1.10",
```

Comment:
Nice catch, using `Google\Cloud\Storage` in our service definition means that we are directly dependent on this.

Reply:
what about `project-base/composer.json` ???